prow/plugins.yaml

```diff
@@ -119,3 +119,6 @@ plugins:
   - reopen
   - lgtm
   - trigger
+
+  GoogleCloudPlatform/k8s-multicluster-ingress:
```

Comment:
If you look at top of this file, the repo need to be configured to send prow webhooks, also send an invitation to @k8s-ci-robot to make it work. Are you the owner of this repo?

Reply:
No instructions at the top of the file, but done and done

Reply:
cc @cjwagner maybe we want to put in some internal docs so that we know where to find different things

Reply:
ALso added GoogleCloudPlatform/k8s-multicluster-ingress to the top of file